### PR TITLE
Rewording some pootle_fs commands messages

### PR DESCRIPTION
**Pull request by unho**

### CODE REVIEW

File: pootle/apps/pootle_fs/management/commands/fs_commands/rm.py [hunk 1]
```diff
@@ -19,4 +19,4 @@ def add_arguments(self, parser):
             "--force",
             action="store_true",
             dest="force",
-            help=("Stage files and stores that are conflicting for removal"))
+            help=("Stage files and/or stores for removal"))
```
Comment (phlax) [inline pootle/apps/pootle_fs/management/commands/fs_commands/rm.py]: this needs to retain the "that are conflicting" and in fact probs needs to have the following added "that are conflicting or untracked"

File: pootle/apps/pootle_fs/management/commands/fs_commands/state.py [hunk 2]
```diff
@@ -27,9 +27,8 @@
 
 
 class StateCommand(FSAPISubCommand):
-    help = (
-        "Show state of tracked and untracked files and Stores for "
-        "specified project")
+    help = ("Show state of tracked and untracked files and Stores for the "
+            "specified project")
     api_method = "state"
```
Comment (phlax) [inline pootle/apps/pootle_fs/management/commands/fs_commands/state.py]: not sure what is being changed here

File: pootle/apps/pootle_fs/management/commands/fs_commands/rm.py (hunk 1 above)
Comment (unho) [inline pootle/apps/pootle_fs/management/commands/fs_commands/rm.py]: ok. I thought that it was possible to remove any file/store independently of conflict/untracked status.

File: pootle/apps/pootle_fs/management/commands/fs_commands/state.py (hunk 2 above)
Comment (unho) [inline pootle/apps/pootle_fs/management/commands/fs_commands/state.py]: Only indent.

Comment (phlax) [inline pootle/apps/pootle_fs/management/commands/fs_commands/state.py]: please dont include unnecessary formatting changes

Comment (unho) [inline pootle/apps/pootle_fs/management/commands/fs_commands/state.py]: ok, will undo.

File: pootle/apps/pootle_fs/management/commands/fs_commands/rm.py (hunk 1 above)
Comment (phlax) [inline pootle/apps/pootle_fs/management/commands/fs_commands/rm.py]: the logic is this
- if a store or file has been removed, the rm command will stage the corresponding resource for removal
- if used with force, it will also stage *any* conflicting stores/files and *any* new stores/files

File: pootle/apps/pootle_fs/management/commands/fs_commands/state.py (hunk 2 above)
Comment (unho) [inline pootle/apps/pootle_fs/management/commands/fs_commands/state.py]: Just noticed that I added a "the". So keeping this.

**Comment by phlax** [comment]
lgtm
